science/nektarpp/Portfile

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               mpi 1.0
+PortGroup               gitlab 1.0
+
+name                    nektarpp
+homepage                https://www.nektar.info/
```

Comment:
These 2 duplicate lines can be removed.

Reply:
I'm not sure I see duplicate lines?

Reply:
Line 9 is the same as line 17. Line 9 should be removed. 

Line 8 will be overridden by the `gitlab.setup` command (line 12). It will need to be moved to after that line to change the name.

Reply:
Ah okay great, I totally didn't see the one on line 17. Have removed the one on line 9 now.